projects/core/src/occ/adapters/cart/occ-cart-voucher.adapter.ts

```diff
@@ -0,0 +1,65 @@
+import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
```

Comment:
This file need to be refactored, all urls come from OccEndpointsService
See https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/4555b2c9b0bea7dc95a28dcf8490b06d7c074513/projects/core/src/occ/adapters/user/occ-user-notification-preference.adapter.ts

Reply:
configuration in :
occ-endpoints.model.ts
default-occ-user-config.ts

Reply:
Done